### PR TITLE
[Fix] PHP warning on (AnsPress) Categories widget

### PR DESCRIPTION
**Pull request by mebishalnapit**
This PR includes:

* Fixes PHP warning message for undefined array key title when using this widget initially
* Fixes PHP warning message for undefined array key hide_empty when using this widget initially
* Fixes PHP warning message for undefined array key parent when using this widget initially
* Fixes PHP warning message for undefined array key number when using this widget initially
* Fixes PHP warning message for undefined array key orderby when using this widget initially
* Fixes PHP warning message for undefined array key order when using this widget initially

### CODE REVIEW

File: addons/categories/widget.php [hunk 1]
```diff
@@ -56,9 +70,6 @@ public function widget( $args, $instance ) {
 			'order'      => $instance['order'],
 		);
 
-		$icon_width  = ! empty( $instance['icon_width'] ) ? $instance['icon_width'] : 32;
```
Comment (rahularyan) [inline addons/categories/widget.php:59]: @mebishalnapit Why this and below lines were removed?

**Review by rahularyan — CHANGES_REQUESTED** [review]

File: addons/categories/widget.php (hunk 1 above)
Comment (mebishalnapit) [reply addons/categories/widget.php:59]: @rahularyan These are now being set from here: https://github.com/anspress/anspress/pull/606/files/573228d38c65e3b8674f013d474fc2ad6029f668#diff-d9aa7d778f6a9e5c7cb06e952a0a2d4e59723416eb6f51ac7fdc93ef0aa81a97R40-R53 and hence not needed anymore so they were deleted.

Comment (rahularyan) [reply addons/categories/widget.php:59]: You are setting default value there. What if those args are set as an empty string or 0?

Comment (mebishalnapit) [reply addons/categories/widget.php:59]: Maybe adding those values as default might be better option to display the icons. If not then, I'll remove them and make those two value to be set as needed, i.e., same as previous.

**Pull request closed without merging**